roles/drupal10/molecule/default/converge.yml

```diff
@@ -11,10 +11,22 @@
     - running_on_server: false
     - install_ruby_from_source: true
   pre_tasks:
-    - name: update cache
-      apt:
+    - name: Update cache
+      ansible.builtin.apt:
+        name: locales
         update_cache: true
         cache_valid_time: 600
+
+    - name: Ensure en_US.UTF-8 locale is uncommented in /etc/locale.gen
+      ansible.builtin.lineinfile:
+        path: /etc/locale.gen
+        regexp: '^# en_US.UTF-8 UTF-8'
+        line: 'en_US.UTF-8 UTF-8'
+        state: present
```

Comment:
Oh, I see, these changes are not related to the syslog work, just incidental test fixes?